### PR TITLE
Adding rtbdemand new adapter

### PR DESCRIPTION
**Pull request by rtbdemand**
## Type of change
<
!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [X ] New bidder adapter   
- 
## Description of change

Added new Bidder adapter

- test parameters for validating bids
{
                        bidder:'rtbdemand',
                        params: {
                            //Publisher ID
                            zoneid: '9999',
                            floor: 0.05,
                            server: 'bidding.rtbdemand.com'
                        }
                    }

using zoneid 9999 for test responses

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

Yes already tested
- contact email of the adapter’s maintainer
- [ X] official adapter submission
rtb@rtbdemand.com

## Other information

Shreyans Chopra from Rtbdemand.com
shreyanschopra@rtbdemand.com

### CODE REVIEW

**Comment by rtbdemand** [comment]
Also once added please let me know how to include this in http://prebid.org/download.html

**Comment by rtbdemand** [comment]
Adding new bidder rtbdemand

**Comment by rtbdemand** [comment]
Adding new bidder

**Comment by rtbdemand** [comment]
Can you please get this added

File: modules/rtbdemandBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.zoneid);
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    return validBidRequests.map(bidRequest => {
+      var server = bidRequest.params.server || BIDDER_SERVER;
+      console.log(JSON.stringify(bidderRequest));
+      console.log(JSON.stringify(bidderRequest.bids[0].sizes[0]));
```
Comment (dbemiller) [inline modules/rtbdemandBidAdapter.js]: Prebid doesn't `console.log` anything unless it's being run in debug mode.

Use `utils.logMessage()` for these instead

File: test/spec/modules/rtbdemandBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+          height: 90,
+          ad: '<!-- Creative -->',
+          netRevenue: true,
+          currency: 'USD',
+          ttl: 360,
+        }
+      ];
+
+      let result = spec.interpretResponse({ body: response });
+      console.log(JSON.stringify(result));
```
Comment (dbemiller) [inline test/spec/modules/rtbdemandBidAdapter_spec.js]: Tests shouldn't `console.log` anything at all

**Comment by rtbdemand** [comment]
Ok I have removed the console statments from all files .Also now the response is updated to not have broken url

**Comment by dbemiller** [comment]
@rtbdemand Thanks for removing the `console.logs`... but it looks like there were some unintended changes too.

There's now a pair of duplicate files: `test/spec/rtbdemandBidAdapter_spec` _and_ `test/spec/modules/rtbdemandBidAdapter_spec`.

Only the `test/spec/modules` one should exist.

**Comment by rtbdemand** [comment]
Yes remove the file now test/spec/rtbdemandBidAdapter_spec.js

**Pull request merged**